app/jenkins-for-jira-ui/src/components/ConnectionWizard/ConnectionWizard.tsx

```diff
@@ -76,6 +76,7 @@ const ConnectionWizard = (): JSX.Element => {
 				appearance="primary"
 				className={cx(connectionWizardButton)}
 				onClick={(e) => handleNavigateToServerNameScreen(e)}
+				testId="team-gathered"
```

Comment:
/big nit Could we add a more descriptive test-id here? maybe `connection-wizard-team-gathered-btn`